test/clj/game_test/cards/agendas.clj

```diff
@@ -158,7 +158,20 @@
       (prompt-select :corp kati)
       (is (empty? (:prompt (get-runner))) "Fall Guy prevention didn't occur")
       (is (= 1 (count (:discard (get-runner)))) "Kati Jones trashed"))))
-	  
+
+(deftest city-works-project
+  ;; City Works Project - do 2 + advancement counters meat damage on access
+  (do-game
+    (new-game (make-deck "Weyland Consortium: Building a Better World" [(qty "City Works Project" 1)])
```

Comment:
You don't need to make sure ID agrees with agenda for the tests. Tests care not for legal decks ;)

Reply:
That's great to know.